include/seqan/sequence/adapt_stl_container.h

```diff
@@ -585,6 +585,10 @@ capacity(std::array<TChar, N> const & me)
 // Function empty
 // ----------------------------------------------------------------------------
 
+// VC2015 implements some C++17 functions which would collide for 
+// applications that do using namespace std
+// http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2014/n4280.pdf
+#if _MSC_VER < 1900
```

Comment:
I did not find an option in vs to set a specific c++ standard, looks like other people had this problem too:
http://stackoverflow.com/questions/23386472/disable-c11-features-in-vs2013